src/api-reference/accounting-integration/v4.accountingintegration-get-started.markdown

```diff
@@ -47,7 +47,7 @@ SAP Concur users must purchase Concur Expense or Concur Invoice in order to use
 * [Financial Integration Service v4 (FIS)](https://developer.concur.com/api-reference/financial-integration/v4.financial-integration.html)
 * [Events Subscription Service (ESS)](https://developer.concur.com/api-reference/common/ess/getting-started.html)
 * [List v4](https://developer.concur.com/api-reference/common/lists/v4.list.html)
-* [Image v3](https://developer.concur.com/api-reference/image/v3.image.html) (to support image posting)
+* [Image v1](https://developer.concur.com/api-reference/image/v1.image.html) (to support image posting)
```

Comment:
Is this really what we want to do? Image v1 has come up quite a number of times lately as a question from partners because it's deprecated.

Reply:
Yes. This is on purpose. Feel free to reach out internally if you wish to discuss this point further (This comment is on a public platform)